src/gameobjects/graphics/Graphics.js

```diff
@@ -548,6 +548,110 @@ var Graphics = new Class({
         return this;
     },
 
+    /**
+     * Fill a rounded rectangle with the given position, size and radius.
+     *
+     * @method Phaser.GameObjects.Graphics#fillRoundedRect
+     * @since 3.11.0
+     *
+     * @param {number} x - The x coordinate of the top-left of the rectangle.
+     * @param {number} y - The y coordinate of the top-left of the rectangle.
+     * @param {number} width - The width of the rectangle.
+     * @param {number} height - The height of the rectangle.
+     * @param {number} [radius = 20] - The corner radius; It can also be an object to specify different radii for corners
+     * @param {number} [radius.tl = 0] Top left
+     * @param {number} [radius.tr = 0] Top right
+     * @param {number} [radius.br = 0] Bottom right
+     * @param {number} [radius.bl = 0] Bottom left
+     *
+     * @return {Phaser.GameObjects.Graphics} This Game Object.
+     */
+    fillRoundedRect: function (x, y, width, height, radius)
+    {
+        if (typeof radius === 'number')
+        {
+            radius = {tl: radius, tr: radius, br: radius, bl: radius};
+        }
+        else if (typeof radius === 'object')
+        {
+            var defaultRadius = {tl: 0, tr: 0, br: 0, bl: 0};
+            for (var side in defaultRadius)
+            {
+                radius[side] = radius[side] || defaultRadius[side];
+            }
+        }
+        else
+        {
+            radius = {tl: 20, tr: 20, br: 20, bl: 20};
```

Comment:
The above condition defines different default values than this. Perhaps one default should be define and referenced in each case?

Reply:
I was thinking and changing it one way and another before making a pull request and then decided it for this way. It is a bit confusing, but I think that if you want rounded rectangle, then the corners should be rounded if you omit the parameter. But if you specify a parameter for at least one of the corners, then you have already set it as rounded and maybe you don't want others to be rounded - in that case you don't have to set all other parameters to 0. If it's too confusing, then we can make the default 20 in any case.

Reply:
That's a valid point, I think it makes sense to keep it the way you have it then :)

However, you may want to consider adding the default radius in the constructor and then update `setDefaultStyles` to accept an override.

Reply:
setDefaultStyles sets fill and stroke style that each Graphics object uses. I don't think default radius for one specific Graphics type should be there. Otherwise Ellipse's smoothness would probably also be there already.